Tests/MapboxNavigationTests/NavigationMapViewTests.swift

```diff
@@ -47,7 +47,7 @@ class NavigationMapViewTests: TestCase {
         ])
         
         XCTAssertEqual(congestionSegments.count, 1)
-        XCTAssertEqual(congestionSegments[0].0.count, 10)
+        XCTAssertEqual(congestionSegments[0].0.count, 6)
```

Comment:
This is caused by [the change in merging method](https://github.com/mapbox/mapbox-navigation-ios/pull/3603/files#diff-16778b2708d85c9a45c23688c69727bff4f569fb3f240cf93618380dbe262663R92). Previously it would duplicate bordering points. For example when merging segments with coords `1 -> 2 -> 3` it would result in `[1, 2, 2, 3]`. With change above it would result in `[1,2,3]`.
I didn't notice changes in resulting route lines drawing, but TBH I didn't test it too much. Does anybody knows if such merging was intended.